google/cloud/aiplatform/__init__.py

```diff
@@ -48,25 +48,21 @@
 """
 init = initializer.global_config.init
 
-log_param = metadata.metadata_service.log_param
 log_params = metadata.metadata_service.log_params
-log_metric = metadata.metadata_service.log_metric
 log_metrics = metadata.metadata_service.log_metrics
-set_experiment = metadata.metadata_service.set_experiment
 get_experiment = metadata.metadata_service.get_experiment
+get_pipeline = metadata.metadata_service.get_pipeline
 set_run = metadata.metadata_service.set_run
```

Comment:
set_run might need to be called start_run. Can you check with Nate and update accordingly if needed.